chess/svg.py

```diff
@@ -187,7 +187,7 @@ def board(board: Optional[chess.BaseBoard] = None, *,
     :param flipped: Pass ``True`` to flip the board.
     :param coordinates: Pass ``False`` to disable the coordinate margin.
     :param lastmove: A :class:`chess.Move` to be highlighted.
-    :param check: A square to be marked as check.
+    :param check: A :class:`chess.Square` to be marked as being under check.
```

Comment:
The reason I omitted the "~" was because the `chess.Move` and the other previous parameters have been written with the "chess." coming before it. It should follow the convention I believe.

Reply:
I see. Yeah, that makes sense.

I am thinking also about the `to be marked as being under check` part of the sentence. I don't think a square can be `under check` but rather `in check`.

The Wikipedia article `Check (chess)` says this: _A check is a condition in chess [...] that occurs when a player's king [...] **is under threat of capture** on their opponent's next turn. A king so threatened is said to be **in check**._

But the article talks about the king, so a chess piece. I am now thinking that a square can't be _in check_ per se, but a chess piece on that square can (if it's a king, of course).

Maybe write it like this: `A :class:chess.Square to be marked indicating check.`